src/scenario-generator/template.ts

```diff
@@ -0,0 +1,55 @@
+type TemplateContract = {
+  ofConfig(): string;
+};
+
+class ScenarioTemplate implements TemplateContract {
+  ofConfig(): string {
+    return `import type { TScenarioConfig } from '@dataai/msw-auto-mock';
```

Comment:
```suggestion
    return `import type { TScenarioConfig } from './__types__/scenarios/scenarios.type';
```
이렇게 import해야 할 것 같습니다!

Reply:
14f3702ff6f0f605f24cd8b7da46cf4fb7f458e9 패키지 이름 변경하고, e9d927f4eb568768aeacd42fe13a0a14ce4080f6 package.json에 있는 패키지 이름 직접 참조해서 사용하도록 변경했습니다~!